arb_os/gasAccounting.mini

```diff
@@ -161,12 +158,8 @@ public impure func gasAccounting_init() {    // initialize on boot, before we ha
     setGasAccountingParamsFromSpeedLimit(const::DefaultSpeedLimitPerSecond);
 
     let pricingParams = struct {
-        l1GasPerL2Tx: const::Charging_DefaultL1GasPerL2Tx,
-        arbGasPerL2Tx: const::Charging_DefaultArbGasPerL2Tx,
         l1GasPerL1CalldataUnit: const::Charging_DefaultL1GasPerCalldataUnit,
-        arbGasPerL1CalldataUnit: const::Charging_DefaultArbGasPerCalldataUnit,
         l1GasPerStorage: const::Charging_DefaultL1GasPerStorage,
-        arbGasPerStorage: const::Charging_DefaultArbGasPerStorage,
```

Comment:
Looks like some/all of these constants may be unused after this. Would be good to remove them, and also to have @RachelFranks's PR that would warn on these merged

Reply:
Removed the unused constants.